components/configuration-accessor/default/package.mjs

```diff
@@ -16,51 +20,104 @@ const isPrefixArray = (prefix, array) => {
   }
 };
 
-export const generateParseSource = (name) => {
-  const regexps = [
-    new RegExp(`^(?<before>${name})(?<after>($|\\s[\\s\\S]*$))`, "u"),
-    new RegExp(`^(?<before>npx\\s+${name})(?<after>($|\\s[\\s\\S]*$))`, "u"),
-    new RegExp(
-      `^(?<before>npm\\s+exec\\s+mocha)(?<after>($|\\s[\\s\\S]*$))`,
-      "u",
+const findPositionalIndex = (tokens, index) => {
+  if (index < tokens.length) {
+    const token = tokens[index];
+    if (token === "-") {
```

Comment:
I don't understand this. Why is `-` special? It could mean standard in or out, as is conventional with unix tools.

Reply:
Indeed. In the case below, `foo.js` should not be understood as a positional argument. We should crash because there is little we can do to hook the command if the script is read from stdin.

```sh
node - foo.js
```